<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTask</class>
 <widget class="QDialog" name="NewTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>487</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>任务名称：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leTaskName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>站点名称：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leSiteName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pbSwitchWebsite">
      <property name="text">
       <string>切换站点</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="leSearchWords">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSearch">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbSearch</sender>
   <signal>clicked()</signal>
   <receiver>NewTask</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
